src/util/index.ts

```diff
@@ -73,6 +75,33 @@ export const wrapText = (
     };
 };
 
+export const hasUrlParamsSettings = () => {
+    const urlSettings = [URL_PARAM_KEY_TIME];
+    const parsed = queryString.parse(location.search);
+    for (let index = 0; index < urlSettings.length; index++) {
+        const paramName = urlSettings[index];
+        if (parsed[paramName] !== undefined) {
+            return true;
+        }
+    }
+    return false;
+};
+
```

Comment:
Bumped this from Modal component into a utility function in case future params also need to be reset, without it I was accidentally concatenating multiple time params when sharing a new timestamp from a URL that already had a param on it.

Reply:
these new functions should have a few tests (there's a test file for this file in the same folder